### PR TITLE
Fix staticcheck issues

### PR DESCRIPTION
**Pull request by Shatakshi0805**
Signed-off-by: Shatakshi <shatakshi.gupta85@gmail.com>
Related to Bugbash.

### CODE REVIEW

File: controller/share_manager_controller.go [hunk 1]
```diff
@@ -398,11 +398,7 @@ func (c *ShareManagerController) isShareManagerRequiredForVolume(volume *longhor
 	// no active workload, there is no need to keep the share manager around
 	hasActiveWorkload := volume.Status.KubernetesStatus.LastPodRefAt == "" && volume.Status.KubernetesStatus.LastPVCRefAt == "" &&
 		len(volume.Status.KubernetesStatus.WorkloadsStatus) > 0
-	if !hasActiveWorkload {
-		return false
-	}
-
-	return true
+	return !(!hasActiveWorkload)
```
Comment (shuo-wu) [inline controller/share_manager_controller.go:401]:
```suggestion
	return hasActiveWorkload
```

**Review by shuo-wu — CHANGES_REQUESTED** [review]

**Comment by jenting** [comment]
@Shatakshi0805 are you willing to keep working on this PR?

**Comment by Shatakshi0805** [comment]
apologies for being inactive here. i have been quite caught up due to college work and exams. i will update the pr with requested changes as soon as possible.

**Comment by shuo-wu** [comment]
> apologies for being inactive here. i have been quite caught up due to college work and exams. i will update the pr with requested changes as soon as possible.

Sure. Thank you! Feel free to ping us after updating this PR

**Pull request closed without merging**